htdocs/core/triggers/interface_50_modAgenda_ActionsAuto.class.php

```diff
@@ -94,7 +94,9 @@ public function runTrigger($action, $object, User $user, Translate $langs, Conf
 
 		$langs->load("agenda");
 
-		if (empty($object->actiontypecode)) {
+		if (strpos($action, 'SENTBYMAIL') && (empty($object->actiontypecode) || $object->actiontypecode == 'AC_OTH_AUTO')) {
+			$object->actiontypecode = 'AC_EMAIL';
```

Comment:
This generates a problem.
It break the rule:
If action is an automatic action, we use a typecode that has a type 'systemauto', if it is a manual action we use a typecode that has a type 'system'. Here we force to AC_EMAIL so manual action whatever is nature of event, so even for automatic action.

What value to you have for $action and in for which situation do you need to fix this ?

Reply:
As discussed during devcamp, i cancel the PR.